slack_sdk/web/client.py

```diff
@@ -1555,34 +1555,53 @@ def emoji_list(self, **kwargs) -> SlackResponse:
         return self.api_call("emoji.list", http_verb="GET", params=kwargs)
 
     def files_comments_delete(
-        self, *, file: str, id: str, **kwargs  # skipcq: PYL-W0622
+        self, *, file: str, id: str  # skipcq: PYL-W0622
     ) -> SlackResponse:
         """Deletes an existing comment on a file.
 
         Args:
             file (str): The file id. e.g. 'F1234467890'
             id (str): The file comment id. e.g. 'Fc1234567890'
         """
-        kwargs.update({"file": file, "id": id})
-        return self.api_call("files.comments.delete", json=kwargs)
+        return self.api_call("files.comments.delete", json={"file": file, "id": id})
 
-    def files_delete(self, *, file: str, **kwargs) -> SlackResponse:
```

Comment:
The `kwargs` is still needed even if we have all the possible arguments. `**kwargs` can work as a workaround when developers try beta features and/or the SDK does not support newly added arguments until new version releases. This request is the same for other changes.